std/ascii.d

```diff
@@ -102,6 +103,70 @@ enum LetterCase : bool
     assert(sha1HMAC == "49f2073c7bf58577e8c9ae59fe8cfd37c9ab94e5");
 }
 
+/++
+    All the control characters in ASCII table
++/
+enum ControlChar : char
+{
+    NUL = '\x00', /// Null
+    SOH = '\x01', /// Start of heading
+    STX = '\x02', /// Start of text
+    ETX = '\x03', /// End of text
+    EOT = '\x04', /// End of transmission
+    ENQ = '\x05', /// Enquiry
+    ACK = '\x06', /// Acknowledge
+    BEL = '\x07', /// Bell
+    BS  = '\x08', /// Backspace
+    TAB = '\x09', /// Horizontal tab
+    LF  = '\x0A', /// NL line feed, new line
+    VT  = '\x0B', /// Vertical tab
+    FF  = '\x0C', /// NP form feed, new page
+    CR  = '\x0D', /// Carriage return
+    SO  = '\x0E', /// Shift out
+    SI  = '\x0F', /// Shift in
+    DLE = '\x10', /// Data link escape
+    DC1 = '\x11', /// Device control 1
+    DC2 = '\x12', /// Device control 2
+    DC3 = '\x13', /// Device control 3
+    DC4 = '\x14', /// Device control 4
+    NAK = '\x15', /// Negative acknowledge
+    SYN = '\x16', /// Synchronous idle
+    ETB = '\x17', /// End of transmission block
+    CAN = '\x18', /// Cancel
+    EM  = '\x19', /// End of medium
+    SUB = '\x1A', /// Substitute
+    ESC = '\x1B', /// Escape
+    FS  = '\x1C', /// File separator
+    GS  = '\x1D', /// Group separator
+    RS  = '\x1E', /// Record separator
+    US  = '\x1F', /// Unit separator
+    DEL = '\x7F' /// Delete
+}
+
+///
+@safe pure nothrow @nogc unittest
+{
+    // Because all ASCII characters fit in char, so do these
+    static assert(ControlChar.ACK.sizeof == 1);
+
+    static assert(ControlChar.NUL == '\0');
+    static assert(ControlChar.BEL == '\a');
+    static assert(ControlChar.BS  == '\b');
+    static assert(ControlChar.FF  == '\f');
+    static assert(ControlChar.LF  == '\n');
+    static assert(ControlChar.CR  == '\r');
+    static assert(ControlChar.TAB == '\t');
+    static assert(ControlChar.VT  == '\v');
```

Comment:
Why aren't you testing all of them?

Reply:
I also have a feeling that there's a mention of more escape sequences existing, but I found nothing else here: https://dlang.org/spec/lex.html#characterliteral

Where can I find the rest, if there are?

Reply:
The control characters are mostly all in a row. I'd probably just loop over them and verify that they equal 0 up through the last one in a row and then test any individually that I'd have to. I probably wouldn't even bother testing the escape sequences. I suppose that testing them in addition to the loop helps ensure that you didn't screw up the loop, but a simple loop with `EnumMembers` should make it easy to test almost all of the values.

Reply:
Done. I left the escape char literal tests in place, though.